web/apps/serve.go

```diff
@@ -75,7 +76,11 @@ func handleIntent(c echo.Context, i *instance.Instance, slug, intentID string) {
 	if !allowed {
 		return
 	}
-	from := i.SubDomain(intent.Client).String()
+	parts := strings.SplitN(intent.Client, "/", 2)
```

Comment:
Works far better ! Thanks @nono !

Still have a warning message from Chrome about the `ALLOW-FROM` directive, but from what I read on the web, it is a known problem.